src/routes/devices/index.ts

```diff
@@ -4,17 +4,26 @@
  **********************************************************************/
 
 import { Router } from 'express'
-import { deviceGetValidator, metadataQueryValidator } from './deviceValidator'
+import { metadataQueryValidator, validator } from './deviceValidator'
 import { disconnect } from './disconnect'
 import { getAll } from './getAll'
 import { stats } from './stats'
 import { get } from './get'
+import { tags } from './tags'
+import { insertDevice } from './create'
+import { updateDevice } from './update'
+import { deleteDevice } from './delete'
+import { param } from 'express-validator'
 
 const deviceRouter: Router = Router()
 
 deviceRouter.get('/', metadataQueryValidator(), getAll)
 deviceRouter.get('/stats', stats)
-deviceRouter.delete('/disconnect/:guid', disconnect)
-deviceRouter.get('/:guid', deviceGetValidator(), get)
+deviceRouter.get('/tags', tags)
+deviceRouter.get('/:guid', param('guid').isUUID(), get)
```

Comment:
Let's call the function getDevice instead of get to match the others

Reply:
done